content/docs/command-reference/exp/index.md

```diff
@@ -37,6 +38,7 @@ positional arguments:
     push                Push a local experiment to a Git remote.
     pull                Pull an experiment from a Git remote.
     remove              Remove local experiments.
+    init                Initialize experiments.
```

Comment:
Sounds like a requirement (like `dvc init`. How can we better describe what this does for the user? (I need to read the rest of the changes before I can suggest something.)

Reply:
Key phrases:
> helps you quickly get started with experiments
reduces boilerplate
initializing pipeline stages

Reply:
For the help output maybe an even shorter form of the specific desc.

> Codify a project variation and run it as an experiment

p.s. and re "sounds like a requirement (like `dvc init`)" - can we still reconsider the name? 😅